frontend/src/organisation/OrganisationDetailTabs.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div v-if="settings">
```

Comment:
Eh, why should we hide all tabs if we have no settings? That seems rather unusual…